utils/file.go

```diff
@@ -71,6 +71,19 @@ func MkTmpdir() (string, error) {
 	return tempDir, os.MkdirAll(tempDir, os.ModePerm)
 }
 
+func MkTmpFile(path string) (*os.File, error) {
+	if path == "" {
+		path = os.TempDir()
+	}
+	if _, err := os.Stat(filepath.Dir(path)); err != nil {
+		return nil, err
+	}
+	tempFile, err := ioutil.TempFile(filepath.Dir(path), ".tmp-"+filepath.Base(path))
```

Comment:
How about we define a static path under /var/lib/sealer/tmp, which could help us clean garbage easier.